Libraries/src/net/relinc/libraries/application/Bar.java

```diff
@@ -77,6 +91,30 @@ public double getRadius() {
 		return this.diameter / 2;
 	}
 
+	public void parseJSONtoParameters(String input) {
+		JSONObject jsonObject = null;
```

Comment:
Maybe move all this inside the `try` and avoid `null`. `null` should be avoided if possible.
```java
		JSONParser jsonParser = new JSONParser();
 		try {
			JSONObject jsonObject = (JSONObject) jsonParser.parse(input);
                         setParametersJSON(jsonObject);
		} catch (org.json.simple.parser.ParseException e) {
			//TODO throw e
		}
 		
```

Reply:
I ended up moving everything inside of try instead of using optional